app/code/Magento/QuoteGraphQl/Model/Resolver/ShippingAddress/AvailableShippingMethods.php

```diff
@@ -65,13 +66,19 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         $shippingRates = $address->getGroupedAllShippingRates();
```

Comment:
Need to check this behavior on frontend.
How to frontend work with `getGroupedAllShippingRates` method and provide the same error message